Documentation/metrics.md

```diff
@@ -35,6 +35,23 @@ is totally unavailable.
 
 `proposals_committed_total` records the total number of consensus proposals committed. This gauge should increase over time if the cluster is healthy. Several healthy members of an etcd cluster may have different total committed proposals at once. This discrepancy may be due to recovering from peers after starting, lagging behind the leader, or being the leader and therefore having the most commits. It is important to monitor this metric across all the members in the cluster; a consistently large lag between a single member and its leader indicates that member is slow or unhealthy.
 
+### disk
+
+These metrics describe the status of the disk operations.
+
+All these metrics are prefixed with `etcd_disk_`.
+
+| Name                               | Description                                           | Type      |
+|------------------------------------|-------------------------------------------------------|-----------|
+| wal_fsync_duration_seconds         | The latency distributions of fsync called by wal      | Histogram |
+| backend_commit_duration_seconds    | The latency distributions of commit called by backend.| Histogram |
+
+A `wal_fsync` is called when etcd persists its log entries to disk before applying them.
+
+A `backend_commit` is called when etcd does an incremental snapshot by commiting an in-memory transcation that contains the most recent changes to on-disk B+Tree.
+
+Abnormally high disk operation duration (`wal_fsync_duration_seconds` or `backend_commit_duration_seconds`) indicates disk issues. It might cause high request complete latency, and a unstable cluster.
```

Comment:
Spikes on `wal_fsync_duration_seconds` or `backend_commit_duration_seconds` often indicates disk issues or an unstable cluster, slowing down the request processing?